pandas/compat/_optional.py

```diff
@@ -85,7 +90,9 @@ def import_optional_dependency(
         * ignore: Return the module, even if the version is too old.
           It's expected that users validate the version locally when
           using ``on_version="ignore"`` (see. ``io/html.py``)
-
+    min_version : str, default None
```

Comment:
can you create an issue (and PR!) to consolidate `raise_on_missing` / `on_version` which really should just be `errors='raise|warn|ignore'`